Create UMA_token_holder_responsibilities.md (#1175)
UMAprotocol/protocol: documentation/oracle/governance/UMA_token_holder_responsibilities.md

> # UMA Token Holder Responsibilities

Reviewer: Add to `map.txt`?

Reviewer 2: I went ahead and did this

1. Voting on price requests from financial contracts depending on the DVM 
1. Governing the UMA ecosystem by voting on parameter changes and approving system upgrades

A tutorial on how to vote on price requests is [here](../tutorials/voting_with_uma_tokens.md).

## Voting on price requests

Owners of UMA tokens will occasionally be asked to vote on price requests from financial contracts in a 2-stage (commit and reveal) process. 
They will have the opportunity to discuss their votes in the #voting channel of the UMA Slack before voting. 

## Governing the UMA ecosystem

Owners of UMA tokens govern 2 areas of the UMA ecosystem: 
1. Financial contracts depending on the DVM
1. The UMA DVM
All governance issues will be addressed via an UMIP process. The UMIP process is detailed [here](./governance/UMIPs.md). 

Because UMA token owners will be responsible for voting on price requests from financial contracts depending on the DVM, they need to approve any price identifiers that they may be asked to opine on and any financial contracts that may create price requests. 
To that end, they need to: 
- Approve new price identifiers

> - Register new contract templates
> - De-register or shut down contract templates

Reviewer: ```suggestion
- Register and de-register contract templates
- Shut down contract instantiations in rare circumstances
```
